test/e2e/util/util.go

```diff
@@ -1506,6 +1508,282 @@ func EnsureGuestWebhooksValidated(t *testing.T, ctx context.Context, guestClient
 	})
 }
 
+func EnsureGlobalPullSecret(t *testing.T, ctx context.Context, mgmtClient crclient.Client, guestClient crclient.Client, entryHostedCluster *hyperv1.HostedCluster) error {
+	//AtLeast(t, Version420)
+
+	var (
+		dummyImageTagMultiarch = "quay.io/hypershift/sleep:multiarch"
+		dummyImageTag12        = "quay.io/hypershift/sleep:1.2.0"
+		err                    error
+
+		// Additional Pull Secret
```

Comment:
why is "additional pull secret" declared as registrySecret and then fetch as UserProvidedPullSecret. Can we please use the same term everywhere when referring to the same thing?

Reply:
also referred as dummy pull secret